src/ucp/api/ucp.h

```diff
@@ -144,7 +144,8 @@ enum ucp_feature {
                                            operations support */
     UCP_FEATURE_WAKEUP = UCS_BIT(4),  /**< Request interrupt notification
                                            support */
-    UCP_FEATURE_STREAM = UCS_BIT(5)   /**< Request stream support */
+    UCP_FEATURE_STREAM = UCS_BIT(5),  /**< Request stream support */
+    UCP_FEATURE_X      = UCS_BIT(6)   /**< Request am support */
```

Comment:
@yosefe Any ideas how to handle this in a portable way ? we don't have that many bits available ...

Reply:
maybe have single bit to enable ALL experimental features?

Reply:
@yosefe That is what I was trying to do, I need to fix the comment. Only problem I thought of is that some experimental features may not be available on all systems. For example, someone might want experimental feature A, which enables all of them, but the transport does not support Active Messages, which will throw an error despite the user not wanting active messages.

Reply:
basically all feature should work on all systems.
BTW another option would be to enable the features through environment variable (rather than by a flag for UCP_INIT). At least while it's experimental.
 otherwise need to rename "UCP_FEATURE_X" to "UCP_FEATURE_EXPERIMENTAL"

Reply:
I'm okay with single UCP_FEATURE_EXPERIMENTAL as for now.